src/repositories/FeedbackRepository.ts

```diff
@@ -0,0 +1,56 @@
+import { UserModel } from 'src/models/UserModel';
+import { AbstractRepository, EntityRepository } from 'typeorm';
+
+import { FeedbackModel } from '../models/FeedbackModel';
+import { Uuid } from '../types';
+
+@EntityRepository(FeedbackModel)
+export class FeedbackRepository extends AbstractRepository<FeedbackModel> {
+  public async getAllFeedback(): Promise<FeedbackModel[]> {
+    return await this.repository.createQueryBuilder("feedback")
+    .leftJoinAndSelect("feedback.user", "user")
+    .getMany();
+  }
+
+  public async getFeedbackById(id: Uuid): Promise<FeedbackModel | undefined> {
+    const feedback = await this.repository
+    .createQueryBuilder("feedback")
+    .where("feedback.id=:id", { id })
```

Comment:
decide whether you want to put spaces around the = or not, I see no spaces here and but I see spaces under; stay consistent with style